projects/packages/publicize/tests/php/test-connections-post-field.php

```diff
@@ -254,10 +261,11 @@ public function get_connections() {
 				'facebook' => array(
 					'id_number' => array(
 						'connection_data' => array(
-							'user_id'  => self::$user_id,
-							'id'       => 'test-unique-id456',
-							'token_id' => 'test-unique-id456',
-							'meta'     => array(
+							'user_id'       => self::$user_id,
+							'id'            => '456',
```

Comment:
Do we change this as it's being used now? If so, should these be numbers rather than strings?